shard/src/api/api.py

```diff
@@ -141,12 +142,12 @@ async def settings_access(self, guild, setting=None, value=None):
         return {'value': "unknown setting"}, sc.NOT_FOUND_404
 
     async def tag_autbot_guilds(self, guild_list, user_id: int):
-        all_guilds, _ = await self.bot.manager_client.all_guilds()
+        all_guilds = [guild for guild in await self.bot.manager_client.all_guilds(message.AllGuildsRequest())]
         for guild_dict in guild_list:
             for guild in all_guilds:
-                if int(guild['id']) == int(guild_dict['id']):
+                if guild.id == int(guild_dict['id']):
```

Comment:
is this guaranteed to be an `int` now?

Reply:
Yeah. gRPC will guarantee that we're getting a 64-bit integer